### ISSUE TITLE
Legacy loader fails

### ISSUE THREAD

**Issue by quantumjot** [quantumjot/btrack#340]
May as well remove if not being used?

```suggestion
filtered_idx = [i for i, _ in enumerate(data) if eval(f_eval)]
```

_Originally posted by @paddyroddy in https://github.com/quantumjot/btrack/pull/334#discussion_r1224135831_

**Comment by quantumjot** [comment]
This change needs to be reverted.

**Comment by quantumjot** [comment]
Fixed by #341
